### DISCUSSION TITLE
Error downloading Figma Server Render Image Data

### DISCUSSION THREAD

**Discussion by Danm15** [simonoliver/UnityFigmaBridge#38] [Q&A] [answered]
Ran into this problem while transferring the design from figma,please tell me how to fix it.

<img width="1728" alt="Знімок екрана 2023-11-08 о 16 02 26" src="https://github.com/simonoliver/UnityFigmaBridge/assets/49476308/6c3a8008-d046-4cfd-b3a9-2a4dcda69210">

**Comment by simonoliver** [comment, marked as answer]
Are you using the latest version? (1.0.8). There was an issue with older builds when rendering large numbers of images. Alternatively, try changing the image scale to 1 in the settings (rather than 3).

**Reply by Danm15** [reply]
Good afternoon, I am using version 1.0.8
I did as you said and everything worked, thank you

**Comment by Danm15** [comment]
And one more question, why do objects not appear on the scene immediately after the export from figma, how to fix it?

**Reply by simonoliver** [reply]
Could you explain? If you've set up a prototype flow in Figma it should add the first screen to the stage automatically. Otherwise, it will just create the screens and pages in the library

**Reply by Danm15** [reply]
Got it, thank you
